src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -213,6 +213,7 @@ We refer to the common principles for the standards for describing dates and tim
 | ScaleFactor                          | RECOMMENDED           | [array][] of [numbers][]               | Scale factor for each frame.                                                                                    |
 | ScatterFraction                      | RECOMMENDED           | [array][] of [numbers][]               | Scatter fraction for each frame (Units: 0-100%).                                                                |
 | DecayCorrectionFactor                | RECOMMENDED           | [array][] of [numbers][]               | Decay correction factor for each frame.                                                                         |
+| DoseCalibrationFactor                | RECOMMENDED           | [number][]                             | Value used to calibrate radioactivity count, i.e. transform to meaningful unit (e.g. Becquerel) - value already applied to the data as a multiplication factor that scales them from counts/sec to Bq/ml (dicom tag 0054,1322). |
```

Comment:
I have problems understanding the description. Could you please split this up into sentences with each sentence conveying a meaning? 

Let me clarify what I find confusing:

> Value used to calibrate radioactivity count

--> is "radioactivity count" metadata that corresponds to some already existing field? If yes, can you reference that one, please?

>  - value already applied to the data as a multiplication factor that scales them from counts/sec to Bq/ml (dicom tag 0054,1322).

does that connect to the previous sentence? Or is it a standalone statement? Or an example / clarification?


Lastly:

- We have agreed not to use latin abbreviations in the BIDS specification, see: https://the-turing-way.netlify.app/community-handbook/style.html#avoid-latin-abbreviation --> so please remove/replace the latin
- The table pipes of this table need to be aligned (currently not the case, hence CIs are failing)

Reply:
Thanks for pointing this out @sappelhoff. I have now updated the description, so it is very much in line with the dicom description, and then referring to the dicom tag. Then it is possible to make a dicom lookup, if one wants to search for more information. This also removes the latin abbreviations, and then I have aligned the table pipes.